.editorconfig

```diff
@@ -0,0 +1,19 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+# Change these settings to your own preference
```

Comment:
Could you please remove that comment, all rules you set are fully satisfy this project style, so nothing require to change.